personal-portfolio/js/modal.js

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable id-length */
+/* eslint-disable no-magic-numbers */
+/* eslint-disable max-statements */
+/* eslint-disable func-style */
+/* eslint-disable max-len */
+/* eslint-disable sort-keys */
+
+const projectList = {
+  'firstProject': {
+    'image': {
+      'prj-img': 'images/mobile-popup.png',
+      'altText': 'This image is a project thumbnail',
+      'prj-img-1': 'images/mobile-popup.jpg'
+    },
+    'projectHeading': 'Multi Post Stories: Gain + Glory',
+    'popupHeading': 'Keeping track of hundreds of components',
+    'description':
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    'languages': [
+'Ruby on Rails',
+'CSS',
+'JavaScript',
+'HTML'
+],
+    'projectButton': 'See Project',
+    'liveVersion': 'https://mprotic123.github.io/personal-portfolio',
+    'sourceCode': 'https://github.com/mprotic123'
+  },

```

Comment:
- [ ] Nice job on adding the modals, but there is a minor issue in 👉 [your modal is not correctly displaying the icons ](https://user-images.githubusercontent.com/61562626/128497921-df6aee4d-4004-4009-8843-361895737592.jpeg), and the [ORIGINAL](https://user-images.githubusercontent.com/61562626/128498082-0b99b903-32b2-4be6-8f55-5de853ac6af8.jpeg).

Reply:
your text is floating out of the box

Check the screenshot below and fix this
<img width="314" alt="Screenshot 2021-08-06 at 12 58 55" src="https://user-images.githubusercontent.com/34798898/128507418-a0a03758-8a36-49f2-9d29-1d57a3365650.png">

Reply:
## [OPTIONAL]
* Sorry that these issues weren't pointed out by previous reviewers 🙏  that is why it is marked as OPTIONAL
* But i strongly suggest you fix these as it would make your project look really professional and eye-catching  
* Your modals still have some issues the mobile and desktop view looks different from the original design
* ORIGINAL : 
![image](https://user-images.githubusercontent.com/52497246/128514046-dfee80bf-4240-4502-bbd7-35c231abbd47.png)
![image](https://user-images.githubusercontent.com/52497246/128514075-b44ede77-4082-447f-85a8-1d6b9f47c65a.png)
* YOURS : 
* Kindly fix the error in the screen shot 🙏 
![image](https://user-images.githubusercontent.com/52497246/128514121-c5e74971-949b-46dc-974c-efbfb76c1ff3.png)
![image](https://user-images.githubusercontent.com/52497246/128514151-040421cb-5fcd-4c2c-a444-472a3ba3527b.png)